Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm

```diff
@@ -741,26 +745,20 @@ static NSHTTPCookieAcceptPolicy httpCookieAcceptPolicy(CFHTTPCookieStorageRef co
     if (!m_subscribedDomainsForCookieChanges)
         m_subscribedDomainsForCookieChanges = adoptNS([[NSMutableSet alloc] init]);
     else if ([m_subscribedDomainsForCookieChanges containsObject:(NSString *)host])
-        return;
+        return false;
```

Comment:
This should return true.
`m_subscribedDomainsForCookieChanges` is shared by all `CookieChangeObserver` (i.e. all `NetworkConnectionToWebProcess` / web processes). If we return false here, only the first observer can subscribe.